code/game/gamemodes/miniantags/tourist/tourist_arrivals.dm

```diff
@@ -74,13 +74,14 @@
 			M.change_gender(FEMALE)
 		set_appearance(M)
 		M.equipOutfit(T.tourist_outfit)
-		M.mind.special_role = SPECIAL_ROLE_TOURIST
 		GLOB.data_core.manifest_inject(M)
+		M.mind.special_role = SPECIAL_ROLE_TOURIST
```

Comment:
can you add a comment specifying why this has to come after adding them to the manifest?